unit_tests/test_tlslite_utils_tripledes.py

```diff
@@ -0,0 +1,326 @@
+# Copyright (c) 2018, Adam Varga
+# This is Known Answers Tests file for python_tripledes,py,
+# which implements 3DES cipher in CBC mode
+#
+# CAVP use 3 Keying Options:
+# KO1: KEY1 != KEY2 != KEY3
+# KO2: (KEY1 = KEY3) != KEY2
+# KO3: KEY1 = KEY2 = KEY3
+#
+# See the LICENSE file for legal information regarding use of this file.
+
+# compatibility with Python 2.6, for that we need unittest2 package,
+# which is not available on 3.3 or 3.4
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+from tlslite.utils.python_tripledes import Python_TripleDES
+from tlslite.utils.cryptomath import *
+from tlslite.errors import *
+
+class Test3DES_KATs_KO3(unittest.TestCase):
+    # KATs from the official CAVP
+    # work with one block messages, all in KO3
+
+    #def setUp(self, key, IV):
+    #    self.tripleDes = Python_TripleDES(key, IV)
+
+    def test_3des_vartext_encrypt(self):
+        #Variable Plaintext Known Answer Test, encrypt one block.
+
+        key = bytearray(b'\x01\x01\x01\x01\x01\x01\x01\x01' +
+                        b'\x01\x01\x01\x01\x01\x01\x01\x01' +
+                        b'\x01\x01\x01\x01\x01\x01\x01\x01')
+        IV = b'\x00\x00\x00\x00\x00\x00\x00\x00'
+
+        self.tripleDes = Python_TripleDES(key, IV)
+
+        self.assertEqual(
+            self.tripleDes.encrypt(b'\x80\x00\x00\x00\x00\x00\x00\x00'),
+                                   b'\x95\xf8\xa5\xe5\xdd\x31\xd9\x00')
+
+
+    def test_3des_vartext_decrypt(self):
+        #Variable Plaintext Known Answer Test, decrypt one block
+
+        key = bytearray(b'\x01\x01\x01\x01\x01\x01\x01\x01' +
+                        b'\x01\x01\x01\x01\x01\x01\x01\x01' +
+                        b'\x01\x01\x01\x01\x01\x01\x01\x01')
+        IV = b'\x00\x00\x00\x00\x00\x00\x00\x00'
+
+        self.tripleDes = Python_TripleDES(key, IV)
+
+        self.assertEqual(
+            self.tripleDes.decrypt(b'\x95\xf8\xa5\xe5\xdd\x31\xd9\x00'),
+                                   b'\x80\x00\x00\x00\x00\x00\x00\x00')
```

Comment:
nit: incorrect indentation, `b'\x80\x00\x00\x00\x00\x00\x00\x00'` should be on the same level as `self.tripleDes`

Reply:
self.assertEqual(
            tripleDes.decrypt(b'\x95\xf8\xa5\xe5\xdd\x31\xd9\x00'),
            b'\x80\x00\x00\x00\x00\x00\x00\x00')

According to "test_tlslite_utils_rsakey.py", is this really correct?